python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -179,9 +179,6 @@ def getTargetList(parsedCmd):
         """
         return [parsedCmd.butler]
 
-    def precall(self, parsedCmd):
-        return True
-
```

Comment:
Note that this will also lead to config-checking.  Will that cause a `--clobber-config` to be necessary when doing multiple iterations with different configs, or is that no longer how things work?

Reply:
So this is for the output products, which should only be run once, so that's not a problem.  (Well, you can run it more than once, but then you would need to clobber which is intended).  The fit cycle has its own config checking code to deal with that issue, and that's still in place.